web/components/proposal-table.tsx

```diff
@@ -91,12 +91,7 @@ const columns: ColumnDef<ProposalListParserOutcome[number]>[] = [
       </div>
     ),
     enableHiding: false,
-  },
-  {
-    accessorKey: 'category',
-    filterFn: (row, value) => {
-      return value.includes(row.original.category);
-    },
```

Comment:
This removes the ability to filter by proposal category.
Was this wanted? @robinbb